op-batcher/flags/flags.go

```diff
@@ -99,6 +99,31 @@ var (
 			return nil
 		},
 	}
+	CompressionAlgoFlag = &cli.StringFlag{
+		Name:    "compression-algo",
+		Usage:   "The compression algorithm to use. Valid options: zlib, brotli ",
+		EnvVars: prefixEnvVars("COMPRESSION_ALGO"),
+		Value:   "zlib",
+		Action: func(c *cli.Context, algo string) error {
+			if algo != "zlib" && algo != "brotli" {
+				return fmt.Errorf("unsupported compression algorithm: %s", algo)
+			}
+			return nil
+		},
+	}
+	CompressLevelFlag = &cli.IntFlag{
```

Comment:
Suggest renaming Compress->Compression to be consistent with how you named the previous flag ("compression-algo")